test-cypress/integration/help/system-information.spec.js

```diff
@@ -0,0 +1,120 @@
+                .should('have.class', 'active');
+            // And should have 3 groups
+            getTabContent().find('.application-info-body').closest('.tab-pane')
+                .should('have.class', 'active')
+                .find('.list-group-item').as('groups')
+                .should('have.length', 3);
+
+            getGroupItemLabel(0, '.gdb-item').should('be.visible').and('contain', 'GraphDB');
+            getGroupItemValue(0, '.gdb-item').should('be.visible');
+
+            getGroupItemLabel(0, '.rdf4j-item').should('be.visible').and('contain', 'RDF4J');
+            getGroupItemValue(0, '.rdf4j-item').should('be.visible');
+
+            getGroupItemLabel(0, '.connectors-item').should('be.visible').and('contain', 'Connectors');
+            getGroupItemValue(0, '.connectors-item').should('be.visible');
+
+            getGroupItemLabel(1, '.os-item').should('be.visible').and('contain', 'OS');
+            getGroupItemValue(1, '.os-item').should('be.visible');
+
+            getGroupItemLabel(1, '.java-item').should('be.visible').and('contain', 'Java');
+            getGroupItemValue(1, '.java-item').should('be.visible');
+
+            getGroupItemLabel(1, '.heap-item').should('be.visible').and('contain', 'Heap memory');
+            getGroupItemValue(1, '.heap-item').should('be.visible');
+
+            // Server report help should be expandable
+            getGroupItem(2, '.server-report-help').should('not.be.visible');
+            getGroupItem(2, '.server-report-help-btn').should('be.visible').click();
+            getGroupItem(2, '.server-report-help').should('be.visible');
+            getGroupItem(2, '.server-report-help-btn').should('be.visible').click();
+            getGroupItem(2, '.server-report-help').should('not.be.visible');
+
+            // Report status is visible
+            getGroupItem(2, '.report-status').find('.report-none').should('be.visible');
+
+            // New report should be visible and enabled
+            getGroupItem(2, '.reports-toobar').find('.new-report-btn').should('be.visible')
+                .and('not.be.disabled');
+            // Download report should be visible and disabled
+            getGroupItem(2, '.reports-toobar').find('.download-report-btn').should('be.visible')
+                .and('be.disabled');
+        });
+
+        it('JVM Arguments tab', () => {
+            // JVM arguments tab can be opened
+            selectTab(2);
+            // Tab content is visible and there 14 jvm arguments listed
+            getTabContent().find('.jvm-arguments-body').closest('.tab-pane')
+                .should('have.class', 'active')
+                .find('.list-group-item').as('groups')
+                .should('have.length', 14);
+        });
+
+        it('Configuration parameters tab', () => {
+            // Configuration parameters tab can be opened
+            selectTab(3);
+            // Tab content is visible and there 14 configurations listed
```

Comment:
It's not clear if they are 14 ;)

Anyway, you can test if some of the default keys are present.